Diverse Anpassungen - Siehe Code

jomsuen opened the pull request:
Adjusting design
Refactoring of folders

jomsuen committed:
Refactoring of folders
Adjusting design

schalterDev commented on lib/ui/start_screen/enter_new_room_tab.dart:
```diff
@@ -42,7 +50,19 @@ class EnterNewRoomTab extends StatelessWidget {
+                        children: <Widget>[
+                          Icon(
+                            Icons.vpn_key,
+                            color: Colors.white,
```
Für Farben immer Farben aus dem Theme benutzen. Die Angabe der Farbe ist hier aber unnötig, weil weiß Standart ist

```suggestion
```

schalterDev commented on lib/ui/start_screen/enter_new_room_tab.dart:
```diff
@@ -13,9 +13,17 @@ class EnterNewRoomTab extends StatelessWidget {
               mainAxisAlignment: MainAxisAlignment.spaceBetween,
               children: [
+                Image.asset(
+                  'assets/images/login_jdch.jpg',
```
Am besten immer .png Bilder mit Transparenz verwenden. Ich ändere das Bild gleich und auch diese Zeile

schalterDev committed:
add transparentz to the image

schalterDev committed:
Merge branch 'master' into feature/enter_room_screen

schalterDev commented on lib/ui/start_screen/enter_new_room_tab.dart:
```diff
@@ -1,5 +1,5 @@
+import 'package:quotierte_redeliste/ui/enter_room/enter_room_screen.dart';
 import 'package:quotierte_redeliste/ui/themes/DefaultThemes.dart';
 
 class EnterNewRoomTab extends StatelessWidget {
```
Der Inhalt von diesem Tab muss scrollbar sein, ansonsten sieht man nicht mehr was man eintippt

jomsuen committed:
FAB auf Startscreen angepasst.
enter_room_tab scrollable gemacht.
FAb schiebt tab allerdings noch nicht nach oben

schalterDev committed:
add padding to bottom that fab is not over content

schalterDev committed:
Merge branch 'master' into feature/enter_room_screen

jomsuen committed:
enter_room_screen ein wenig angepasst

schalterDev committed:
Merge branch 'master' into feature/enter_room_screen
